From 5c308603ba792008ee4eccabd9ff53b8ac826ef8 Mon Sep 17 00:00:00 2001
From: Yifeng Zhao <yifeng.zhao@rock-chips.com>
Date: Thu, 12 Nov 2020 19:14:41 +0800
Subject: [PATCH] arm64: configs: rockchip: Enable ACHI

before
    text	   data	    bss	     dec	    hex	filename
15989635	7547498	2289288	25826421	18a1475	vmlinux

after
    text	   data	    bss	     dec	    hex	filename
16139178	7628603	2293512	26061293	18da9ed	vmlinux

Signed-off-by: Yifeng Zhao <yifeng.zhao@rock-chips.com>
Change-Id: I32bfb693d387589fba6b523c2dd8791a32043ff1
---
 arch/arm64/configs/rockchip_defconfig | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 1779c7f99440..a8ab18d0c104 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -269,13 +269,16 @@ CONFIG_BLK_DEV_RAM_SIZE=8192
 CONFIG_BLK_DEV_NVME=y
 CONFIG_SRAM=y
 CONFIG_UID_SYS_STATS=y
-CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_CHR_DEV_SG=y
 CONFIG_CHR_DEV_SCH=y
 CONFIG_SCSI_CONSTANTS=y
 CONFIG_SCSI_LOGGING=y
 CONFIG_SCSI_SCAN_ASYNC=y
+CONFIG_ATA=y
+CONFIG_SATA_AHCI=y
+CONFIG_SATA_AHCI_PLATFORM=y
+# CONFIG_ATA_SFF is not set
 CONFIG_MD=y
 CONFIG_BLK_DEV_DM=y
 CONFIG_DM_CRYPT=y
-- 
2.35.3

